microsoft/PowerToys
# [MWB] fix layout resetting to one row

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #26246


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- change layout to 2-row
- press "refresh connections"
- observe that it no resets to the default layout
